Day2ConsoleAppHomework/Program.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Day2ConsoleAppHomework
 {
+    enum TransactionType { Buy, Sell }
+
     class Program
     {
+        string currency = "DKK";
+
         static void Main(string[] args)
         {
+            string currency = "DKK";
+
+            Console.WriteLine("Input the transaction type:");
+            var userInput = Console.ReadLine();
+            var transactionType = (TransactionType)Enum.Parse(typeof(TransactionType), userInput, true);
+
+            Console.WriteLine("Input the nominal of the trade:");
+            userInput = Console.ReadLine();
+            var nominal = float.Parse(userInput);
+
+            Console.WriteLine("Input the trade price:");
+            userInput = Console.ReadLine();
+            var tradePrice = float.Parse(userInput);
+            
+            Console.WriteLine("Input the trade price for the original trade (only used for sales):");
+            userInput = Console.ReadLine();
+            var origPrice = float.Parse(userInput);
```

Comment:
var origPrice = float.Parse(Console.ReadLine());

Reply:
origPrice is not meaningful

Reply:
Ok. I have no chosen 'originalPrice'. (The other name is inspired by how we would most likely name it in APL).

Reply:
Taking both comments into account, - I did:
 var originalPrice = float.Parse(Console.ReadLine());